DrupalExtended/ruleset.xml

```diff
@@ -0,0 +1,164 @@
+<?xml version="1.0"?>
+<ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="Drupal Extended" xsi:noNamespaceSchemaLocation="../vendor/squizlabs/php_codesniffer/phpcs.xsd">
+  <description>An extension of Drupal coding standards.</description>
+
+  <rule ref="vendor/drupal/coder/coder_sniffer/Drupal">
+    <!-- Exclude in favor of native typehints. See: https://www.drupal.org/project/coding_standards/issues/3238192 -->
+    <exclude name="Drupal.Commenting.VariableComment.MissingVar"/>
+    <!-- Conflicts with "SlevomatCodingStandard.TypeHints.DeclareStrictTypes". -->
+    <exclude name="Drupal.WhiteSpace.OpenTagNewline.BlankLine"/>
+    <!-- Conflicts with "<?php declare(strict_types = 1)." -->
+    <exclude name="Drupal.Commenting.FileComment.Missing"/>
+  </rule>
+
+  <rule ref="vendor/drupal/coder/coder_sniffer/DrupalPractice"/>
+
+  <rule ref="vendor/slevomat/coding-standard/SlevomatCodingStandard">
+    <!-- Excluded rules based on what enabled in 1.x branch. -->
+    <exclude name="SlevomatCodingStandard.Arrays.DisallowImplicitArrayCreation"/>
+    <exclude name="SlevomatCodingStandard.Arrays.MultiLineArrayEndBracketPlacement"/>
+    <exclude name="SlevomatCodingStandard.Arrays.SingleLineArrayWhitespace"/>
+    <!-- Same as 'Squiz.Arrays.ArrayDeclaration.NoCommaAfterLast'. -->
+    <exclude name="SlevomatCodingStandard.Arrays.TrailingArrayComma"/>
+    <exclude name="SlevomatCodingStandard.Attributes.AttributeAndTargetSpacing"/>
+    <exclude name="SlevomatCodingStandard.Attributes.AttributesOrder"/>
+    <exclude name="SlevomatCodingStandard.Attributes.DisallowAttributesJoining"/>
+    <exclude name="SlevomatCodingStandard.Attributes.DisallowMultipleAttributesPerLine"/>
+    <exclude name="SlevomatCodingStandard.Attributes.RequireAttributeAfterDocComment"/>
+    <exclude name="SlevomatCodingStandard.Classes.BackedEnumTypeSpacing"/>
+    <exclude name="SlevomatCodingStandard.Classes.ClassLength"/>
+    <exclude name="SlevomatCodingStandard.Classes.ClassMemberSpacing"/>
+    <exclude name="SlevomatCodingStandard.Classes.ClassStructure"/>
+    <exclude name="SlevomatCodingStandard.Classes.ConstantSpacing"/>
+    <exclude name="SlevomatCodingStandard.Classes.DisallowConstructorPropertyPromotion"/>
+    <exclude name="SlevomatCodingStandard.Classes.DisallowMultiConstantDefinition"/>
+    <exclude name="SlevomatCodingStandard.Classes.DisallowMultiPropertyDefinition"/>
+    <exclude name="SlevomatCodingStandard.Classes.EmptyLinesAroundClassBraces"/>
+    <exclude name="SlevomatCodingStandard.Classes.ForbiddenPublicProperty"/>
+    <exclude name="SlevomatCodingStandard.Classes.MethodSpacing"/>
+    <exclude name="SlevomatCodingStandard.Classes.ParentCallSpacing"/>
+    <exclude name="SlevomatCodingStandard.Classes.PropertyDeclaration"/>
+    <exclude name="SlevomatCodingStandard.Classes.PropertySpacing"/>
```

Comment:
I think it should be enabled with all settings set to 1. This will mean that properties should be wrapped by a single line. Currently, this is a valid code, Drupal sniffers doesn't find problems here:

```php
  /**
   * The node storage.
   */
  protected NodeStorageInterface $nodeStorage;
  /**
   * The node view builder.
   */
  protected NodeViewBuilder $nodeViewBuilder;
```